frontend/src/Components/Exercise.js

```diff
@@ -19,9 +20,22 @@ const Exercise = ({selected}) => {
         setInputVal(e.target.value)
     };
 
+    
+   
+
     const onTry = () => {
+        
         setOutputVal(inputVal);
-        setOutputAnswer(selected.code(...argSplit(inputVal)));
+
+        try {
+            const args = argSplit(selected.arg)
+            const func = Function(args.flat(), selected.hardcode)
```

Comment:
function is declared in trycatch to avoid crashes if the code data is not correct, in case of fail return message in output

*maybe it is a bad idea to declare the function each time when a user clicks the button